engine/battle_anims/functions.asm

```diff
@@ -3912,7 +4024,7 @@ Functionce618:
 	call DeinitBattleAnimation
 	ret
 
-asm_ce61c:
+move_up:
```

Comment:
This sort of label should be local (`.move_up`).

Reply:
With this one and .step_circle they're both called from a previous function

> error: main.asm(438) -> engine/battle_anims/functions.asm(3887): Unknown symbol "Functionce564.step_circle"
error: main.asm(438) -> engine/battle_anims/functions.asm(3887): jr target out of reach (expected -129 < -7027 < 128)

Reply:
I see. `Functionce60a` is between `Functionce5f9` and `move_up`, so `move_up` can't be local to `Functionce5f9`. The underlying issue is that *all* of those should be local labels: the various `call BattleAnim_AnonJumptable`s toward the top of the file have local pointers `.zero`, `.one`, `.two`, etc, but further down they're just `dw FunctionXXXX`. Ideally they would all follow the local `.zero`, etc pattern, when possible. (It won't always be; for example `BattleAnimFunction_PoisonGas` has `dw BattleAnimFunction_SpiralDescent` instead of `dw .one`, and some pieces might be reused by multiple battle anims; but as much as possible.)

Reply:
Done with those and some other unnamed local labels, except for a couple under `BattleAnimFunction_Gust.GustWobble` which I can't understand what they mean